assets/css/login.css

```
@@ -0,0 +1,70 @@
+#add-post {
+    display: none;
+}
+
+.header{
+    position: fixed;
+    top: 0;
+    width: 100%;
+}
+
+.footer{
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+}
+
+.login-container {
+    width: auto;
+    height: 100%;
+    display: flex;
+    align-items: center;
+}
+
+.login-content {
+    text-align: center;
+    border: 2px solid #ccc;
+    padding: 15px 80px 5px 80px;
+    font-size: 1.25em;
+    background-color: #e2e2e2;
+    width: 100%;
+}
```

```suggestion
.login-content {
    text-align: center;
    border: 1px solid #ccc;
    padding: 15px 80px 5px 80px;
    font-size: 1.25em;
    background-color: var(--feedBackgroundColor);
    width: 100%;
    box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px;
    border-radius: 15px;
}
```

1) Use the color variable to keep the palette of the home page
2) 2px border was too much imo
3) Rounded corners look a little better imo
4) box shadow just makes it prettier